linera-execution/src/runtime.rs

```diff
@@ -918,6 +926,25 @@ impl<UserInstance> BaseRuntime for SyncRuntimeHandle<UserInstance> {
     }
 }
 
+/// An extension trait to determine in compile time the different behaviors between contract and
+/// services in the implementation of [`BaseRuntime`].
+trait ContractAndServiceBaseRuntimeDifferences {
```

Comment:
Maybe just `ContractOrServiceRuntime`?
(One could argue that a trait is always about the differences between the different implementations.)

Reply:
Good point, thanks!